### PR TITLE
Assets/abd resize icons

### PR DESCRIPTION
**Pull request by Abdeboskey**
# Som Timer Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
* This PR resizes all of the icons to take up less space, including the 404 notfound image. 
* Adds the final two icons for `Breathwork/Meditation` and `Yoga/Movement` content.
## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

* Manually asserted that assets have the new dimensions in our repository.
* Visual assertion, in the browser.

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- x ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Components/ContentSelection/ContentSelection.js [hunk 1]
```diff
@@ -3,9 +3,10 @@ import style from './ContentSelection.module.scss'
 import { ViewContext } from '../../Context/ViewContext'
 import { VideoContext } from '../../Context/VideoContext'
 import { SettingsContext } from '../../Context/SettingsContext'
+import { getRandomContent } from '../../apiCalls'
 import somaticContentIcon from '../../assets/content/somaticContentIcon.png'
 import meditationContentIcon from '../../assets/content/meditationContentIcon.png'
-import { getRandomContent } from '../../apiCalls'
+import yogaContentIcon from '../../assets/content/yogaContentIcon.png'
```
Comment (sienna-kopf) [inline src/Components/ContentSelection/ContentSelection.js:9]: Yay! Cant wait to see what you came up with for this Yoga Icon!

File: src/Components/PageNotFound/PageNotFound.module.scss [hunk 2]
```diff
@@ -7,7 +7,8 @@
 }
 
 .warning {
```
Comment (sienna-kopf) [inline src/Components/PageNotFound/PageNotFound.module.scss:9]: Getting my feet into some CSS today was a good reminder of how much work goes into making this all look just right!

**Review by sienna-kopf — APPROVED** [review]

File: src/Components/ContentSelection/ContentSelection.js [hunk 3]
```diff
@@ -71,7 +72,14 @@ const ContentSelection = () => {
             aria-label="yoga/movement"
             name="MOVEMENT"
             onClick={fetchVideo}
-          ></button>
+          >
```
Comment (rwilliams659) [inline src/Components/ContentSelection/ContentSelection.js:75]: Yay! So glad to have all the icons in the content selection page now

**Review by rwilliams659 — APPROVED** [review]
Thanks so much for optimizing all the assets - this is gonna help performance & loading speed a TON!

File: src/Components/ContentSelection/ContentSelection.js [hunk 4]
```diff
@@ -55,11 +56,11 @@ const ContentSelection = () => {
             aria-label="breathwork/meditation"
             name="MEDITATION"
             onClick={fetchVideo}
-            >
+          >
             <img
               className={style.meditationIcon}
               src={meditationContentIcon}
-              alt="human sitting in meditation pose"
+              alt="woman sitting in meditation pose"
```
Comment (jkwest-93) [inline src/Components/ContentSelection/ContentSelection.js:63]: I love this icon!! And visible gender representation.

**Review by jkwest-93 — APPROVED** [review]

**Comment by jkwest-93** [comment]
This is really awesome! It's been really cool to see what modifications are necessary in order for our service worker to cache images like we expect.

**Pull request merged**